core/src/main/scala/org/apache/spark/status/api/v1/SimpleDateParam.scala

```diff
@@ -25,8 +25,18 @@ import javax.ws.rs.core.Response.Status
 import scala.util.Try
 
 private[v1] class SimpleDateParam(val originalValue: String) {
+
   val timestamp: Long = {
-    SimpleDateParam.formats.collectFirst {
+    val formats: Seq[SimpleDateFormat] = {
```

Comment:
OK... but this seems like a very complicated way of saying, "parse with one format, and if it fails, parse with the other. Why not just boil this down while we're here? you can even avoid allocating at least one of the two objects in most cases.